apps/worker/src/app/workflow/usecases/send-message/send-message-type.usecase.ts

```diff
@@ -1,12 +1,34 @@
-import { CreateExecutionDetails } from '@novu/application-generic';
-import { MessageEntity, MessageRepository } from '@novu/dal';
+import { CreateExecutionDetails, DetailEnum } from '@novu/application-generic';
+import { DeliveryLifecycleState, MessageEntity, MessageRepository } from '@novu/dal';
 import { SendMessageChannelCommand } from './send-message-channel.command';
 
-export type SendMessageResult = {
-  status: 'success' | 'failed' | 'skipped';
-  reason?: string;
+export enum SendMessageStatus {
+  SUCCESS = 'success',
+  FAILED = 'failed',
+  SKIPPED = 'skipped',
+}
+
+
+
```

Comment:
Biome this file 💪

Reply:
biome stopped working for me :( 
i need to fix it locally